drizzlepac/haputils/catalog_utils.py

```diff
@@ -2846,6 +2846,7 @@ def _evaluate_segmentation_image(self, segm_img, image_data, big_island_only=Fal
         real_pixels = (np.isfinite(image_data) & (image_data != 0)).sum().clip(min=1)
         biggest_source = segm_img.areas.max()/real_pixels
```

Comment:
Need to delete the redundant definition of `biggest_source` on line 2847.  It is replaced by new line 2849.

Reply:
This is what I get for just accepting the change via the GUI which I basically never do - except for this time!

Reply:
I'm sure I could have marked it up correctly in the GUI so it was OK to accept the change!  So it is my fault.